src/components/ChartTodo/ChartTodo.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Paper } from '@material-ui/core';
+import {BarChart,CartesianGrid ,XAxis,YAxis,Bar , ResponsiveContainer  , Tooltip, Text} from 'recharts'
+import classes from './ChartTodo.module.css';
+import axios from '../../axios-ListData';
+import withErrorHandler from '../../hoc/withErrorHandler/withErrorHandler';
+
+
+class ChartTodo extends Component {
+
+
+    state= {
+        todoList : [],
+
+    }
+
+    componentDidMount () {
+        axios.get('/ListTodo.json')
+             .then(response => {
+                let updateTodoData = [] ;
+                let todoData = Object.keys(response.data).map(item => {
+                    return {...response.data[item]  ,id : item}  })
+                    for( let key in todoData) {
+                        updateTodoData.push(todoData[key])
+                    }
+                    const res = Object.values(updateTodoData.reduce((item, {Date}) => {
+                        item[Date] = item[Date] || {Date,count: 0};
+                        item[Date].count++;
+                        return item;
+                      }, Object.create(null)));
```

Comment:
Object.create(null) can be replaced by {},
response.data is an object why to convert it to an array and then back to object